src/contracts/Protocol/ActionCostController.sol

```diff
@@ -29,10 +29,14 @@ contract ActionCostController is Ownable, OnlyApi, IActionCostController {
         actionCost[ACTION.VOTE] = 1e18;
         actionCost[ACTION.CLAIM] = 1e18;
         actionCost[ACTION.CHALLENGE_BURN] = 1000e18;
+        actionCost[ACTION.CHALLENGE_CREATE];
+        actionCost[ACTION.PUBLISH];
 
         actionStage[ACTION.CREATE] = STAGE.CREATE_AND_VOTE;
         actionStage[ACTION.VOTE] = STAGE.CREATE_AND_VOTE;
         actionStage[ACTION.CLAIM] = STAGE.CLAIM_AND_ANSWER;
+        actionStage[ACTION.CHALLENGE_CREATE] = STAGE.REVIEW;
```

Comment:
I wasn't sure if this was correct - If I don't state an actionStage foundry tests fail but not JS interestingly enough

Reply:
Im assuming this is because the JS does not have vault test coverage. This would store the payments for question creation in the vault under REVIEW, meaning funds would currently be lost. We can probably put both under CREATE_AND_VOTE.

Reply:
vault coverage is next on my list :-)

Reply:
updated